tests/e2e/uninstall.go

```diff
@@ -0,0 +1,194 @@
+
+		remainedResources, err := getK8sResources(kubectlOptions,
+			koperatorAvailableResourceKinds,
+			fmt.Sprintf(managedByHelmLabelTemplate, koperatorLocalHelmDescriptor.ReleaseName),
+			"",
+			kubectlArgGoTemplateKindNameNamespace,
+			"--all-namespaces")
+
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(remainedResources).Should(BeEmpty())
+	})
+}
+
+// requireUninstallingZookeeperOperator uninstall Zookeeper-operator Helm chart
+// and remove CRDs.
+func requireUninstallingZookeeperOperator(kubectlOptions k8s.KubectlOptions) {
+	When("Uninstalling zookeeper-operator", func() {
+		requireUninstallingZookeeperOperatorHelmChart(kubectlOptions)
+		requireRemoveZookeeperOperatorCRDs(kubectlOptions)
+	})
+}
+
+// requireUninstallingZookeeperOperatorHelmChart uninstall Zookeeper-operator Helm chart
+// and checks the success of that operation.
+func requireUninstallingZookeeperOperatorHelmChart(kubectlOptions k8s.KubectlOptions) {
+	It("Uninstalling zookeeper-operator Helm chart", func() {
+		uninstallHelmChartIfExists(kubectlOptions, zookeeperOperatorHelmDescriptor.ReleaseName, true)
+		By("Verifying Zookeeper-operator helm chart resources cleanup")
+
+		k8sResourceKinds, err := listK8sResourceKinds(kubectlOptions, "")
+		Expect(err).ShouldNot(HaveOccurred())
+
+		zookeeperAvailableResourceKinds := stringSlicesInstersect(zookeeperCRDs(), k8sResourceKinds)
+		zookeeperAvailableResourceKinds = append(zookeeperAvailableResourceKinds, basicK8sResourceKinds()...)
+
+		remainedResources, err := getK8sResources(kubectlOptions,
+			zookeeperAvailableResourceKinds,
+			fmt.Sprintf(managedByHelmLabelTemplate, zookeeperOperatorHelmDescriptor.ReleaseName),
+			"",
+			kubectlArgGoTemplateKindNameNamespace,
+			"--all-namespaces")
+		Expect(err).ShouldNot(HaveOccurred())
+
+		Expect(remainedResources).Should(BeEmpty())
+	})
+}
+
+// requireRemoveZookeeperOperatorCRDs deletes the zookeeper-operator CRDs
+func requireRemoveZookeeperOperatorCRDs(kubectlOptions k8s.KubectlOptions) {
+	It("Removing zookeeper-operator CRDs", func() {
+		for _, crd := range zookeeperCRDs() {
+			deleteK8sResourceNoErrNotFound(kubectlOptions, defaultDeletionTimeout, crdKind, crd)
+		}
+	})
+}
+
+// requireUninstallingPrometheusOperator uninstall prometheus-operator Helm chart and
+// remove CRDs.
+func requireUninstallingPrometheusOperator(kubectlOptions k8s.KubectlOptions) {
+	When("Uninstalling prometheus-operator", func() {
+		requireUninstallingPrometheusOperatorHelmChart(kubectlOptions)
+		requireRemovePrometheusOperatorCRDs(kubectlOptions)
+	})
+}
+
+// requireUninstallingPrometheusOperatorHelmChart uninstall prometheus-operator Helm chart
+// and checks the success of that operation.
+func requireUninstallingPrometheusOperatorHelmChart(kubectlOptions k8s.KubectlOptions) {
+	It("Uninstalling Prometheus-operator Helm chart", func() {
+		uninstallHelmChartIfExists(kubectlOptions, "prometheus-operator", true)
```

Comment:
**Required**: We **MUST** check potential errors here.

```suggestion
		err := uninstallHelmChartIfExists(kubectlOptions, "prometheus-operator", true)
		
		Expect(err).NotTo(HaveOccurred())
```

Reply:
Fixed in [63ed2cd](https://github.com/banzaicloud/koperator/pull/987/commits/63ed2cdca59f233e329f912c042c39a21ecf3a7c).